rca/enquire_to_study/views.py

```diff
@@ -307,9 +205,8 @@ def set_session_data(self, form_data, enquiry_submission):
 
     def form_valid(self, form):
         country_of_residence = form.cleaned_data["country_of_residence"]
-        if country_of_residence in ["GB", "IE"] and settings.MAILCHIMP_API_KEY:
-            self.post_mailchimp(form.cleaned_data)
-        elif settings.QS_API_PASSWORD:
+
+        if settings.QS_API_PASSWORD:
```

Comment:
This really is the only change. I noticed that Mailchimp won't be used anymore afterwards so I removed them all.